verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -270,7 +273,11 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
                                                     ? new Date(confirmedDate)
                                                     : null,
                                                 notes: c.notes,
-                                                sourceUrl: c.source?.url,
+                                                sourceUrl:
```

Comment:
Do we want a TODO here, or is it captured elsewhere, to make the UI representation a list?

Reply:
I was assuming this would generally be overhauled, since a few things in the UI don't reflect the model anymore (like location != country, age should be a range, etc)